### DISCUSSION TITLE
addressing the backlog of 'needs triage'?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#17946] [General]
Hi NetBox staff, is there any cadence for going back and addressing the issues that still need triage? There are issues going back almost 3 months.

**Comment by jeremystretch** [comment]
No. Unfortunately, there are far more people who submit feature requests than there are who actually contribute to the project, and we no longer have time to address every FR.

You can help free up maintainer time by volunteering to own one of [68 issues currently open](https://github.com/netbox-community/netbox/issues?q=is%3Aissue+is%3Aopen+label%3A%22status%3A+needs+owner%22) that need an owner.

**Reply by ghost** [reply]
shouldn't triage be first in first out though? FR's that are newly created great triaged over FR's that have been waiting for triage for weeks/months.

**Reply by jeremystretch** [reply]
No. We have burned a lot of time in past years on proposals that simply don't go anywhere, and can no longer afford to do so. Other maintainers are free to engage as they see fit, but for my part I've learned to focus my time and effort on proposals that I feel will genuinely benefit the project.

**Reply by ghost** [reply]
maybe it's my misunderstanding of what triage means then.
If a FR hasn't been triaged, are the maintainers even aware of the proposal from that FR that could benefit the project?

I thought the act of triaging a FR meant that the proposal was evaluated, verified it's not a duplicate, a level of effort to perform the work was determined, and/or a determination if the FR would even be implemented at all.

So if a FR hasn't been triaged, I presume the maintainers aren't even aware of it.
